preview/containers/MainContent.jsx

```diff
@@ -41,6 +42,10 @@ class MainContent extends PureComponent {
       const { dispatch } = this.props;
       dispatch(ActionsCreator.updateInvoice(invoiceData));
     });
+    appConfig.watch('profile', (profileData)  => {
```

Comment:
This is very clever. I used `watch` before but totally forgot about it 👍

Anyway, do you think if this is the best place to call `appConfig`? I would like to avoid calling it from too many places, especially inside a component as it might be difficult to track down later. 

I think the only two place that call `appConfig` (from within `mainWindow`) atm is the `middlewares` & some `reducers`. I would like to move all these calls to middlewares, too if possible.

Reply:
Also, should we call `dispose` on the observable before quitting the app?

Reply:
Btw, here's my take  https://github.com/hql287/Manta/pull/163 on the issue

I think this approach feels more like *electron* and it will also handle other changes in the config such as `timeFormat` (or any new field that we might have in the future), not just `logo`. 

What do you think?

Reply:
@hql287 Agree,👍

Reply:
Thanks! :)